_data/universities/cu_boulder.yml

```diff
@@ -0,0 +1,3 @@
+name: "[University of Colorado-Boulder](https://www.colorado.edu/chancellor/newsletter/chancellor/march-11-2020)"
+status: Remote teaching begins Monday, March 16, through remainder of semester
+last_update: 2020-03-11
```

Comment:
I like the idea a lot, especially using different files per school.

What do you think about having more data fields in here for closing start date, closing end date, remote start date, grading period end date, and remote end date (if known)?  That might enable you to do some cool stuff with stats.

Reply:
My immediate reaction thought to that is that everything is changing _daily_ for uni's expected remote-teaching duration, and that there would be no good mechanism for tracking accurate history for an individual school unless someone were updating daily, sometimes hourly. To illustrate, my uni has been sending out daily updates, with each basically escalating the response. My other thought is that the thought of doings stats with jekyll gives me stomach pains.

But maybe just separate fields for start and end would be do-able.

Reply:
You're probably right there.

I'm deferring to @phildini on this one. I'm a +1 though.

Reply:
I just pushed up a few more fields to each of the data files -- `start`, and `expected_end`. Nothing added to the readme rendering yet though. But they're there for others to follow.